examples/simpleReceive.ts

```diff
@@ -0,0 +1,50 @@
+import { OnMessage, OnError, MessagingError, delay, Message, ReceiveMode, Namespace } from "../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = "SERVICEBUS_CONNECTION_STRING";
+const entityPath = "QUEUE_NAME";
+const str = process.env[connectionString] || "";
+const path = process.env[entityPath] || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+let ns: Namespace;
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(str);
+  const client = ns.createQueueClient(path, { receiveMode: ReceiveMode.peekLock, maxConcurrentCalls: 1 });
+  const onMessage: OnMessage = async (brokeredMessage: Message) => {
+    console.log(">>> Message: ", brokeredMessage);
+    console.log("### Actual message:", brokeredMessage.body ? brokeredMessage.body.toString() : null);
+    if (brokeredMessage.body.toString() === "Hello sb world!!") {
+      // console.log(">>>> Calling complete....");
+      // brokeredMessage.complete();
+      // console.log("???????????????????????????????????????????? completed the message....");
+      //brokeredMessage.abandon();
+      console.log("???????????????????????????????????????????? abandoned the message....");
+      // brokeredMessage.defer();
+      // console.log("???????????????????????????????????????????? defered the message....");
+    } else {
+      brokeredMessage.complete();
+    }
+  }
+  const onError: OnError = (err: MessagingError | Error) => {
+    console.log(">>>>> Error occurred: ", err);
+  };
+  //console.log(onMessage, onError);
+  client.receive(onMessage, onError, { autoComplete: true });
+  // const msgs = await client.receiveBatch(10);
```

Comment:
probably worth cleaning up the commented out lines in here and maybe factoring the batch receive out into a separate example

Reply:
Oh yes. Definitely. Will do that.